music21/roman.py

```diff
@@ -4389,6 +4401,24 @@ def test_sevenths_on_alteration(self):
         rn = RomanNumeral('bVII7', 'C', seventhMinor=Minor67Default.CAUTIONARY)
         self.assertEqual(rn.seventh.name, 'A')
 
+    def test_int_figure_case_matters(self):
+        '''https://github.com/cuthbertLab/music21/issues/1450'''
+        minorKeyObj = key.Key("c")
```

Comment:
trying to standardize docstrings as always like:

```
'''
adfahdfkladf
'''
```

even if a one-liner.

The linter should be catching the double quote "c" -- not sure why not here.